articles/user-guide/libraries/machine-learning/intro.md

```diff
@@ -23,14 +23,16 @@ Somewhat similar to traditional methods, quantum classification consists of thre
 - data encoding
 - preparation of a classifier state
 - measurement
-Due to the probabilistic nature of the measurement, these three steps must be repeated multiple times. The measurement may be viewed as a quantum equivalent of non-linear activation.
+Due to the probabilistic nature of the measurement, these three steps must be repeated multiple times. 
 Both the encoding and the computing of the classifier state are done by means of *quantum circuits*. While the encoding circuit is usually data-driven and parameter-free, the classifier circuit contains a sufficient set of learnable parameters. 
 
 In the proposed solution the classifier circuit is composed of single-qubit rotations and two-qubit controlled rotations. The learnable parameters here are the rotation angles. The rotation and controlled rotation gates are known to be *universal* for quantum computation, which means that any unitary weight matrix can be decomposed into a long enough circuit consisting of such gates.
 
+In the proposed version only one circuit followed by a single frequency estimation is supported, which makes the solution a quantum analog of support vector machine with low-degree polynomial kernel.
```

Comment:
```suggestion
In the proposed version, only one circuit followed by a single frequency estimation is supported.
Thus, the solution is a quantum analog of a support vector machine with a low-degree polynomial kernel.
```

This could also be a good place to refer further resources on the kernel trick, since we use it implicitly quite a bit in this argument.